src/modules/layout/components/UserBottomNav.tsx

```diff
@@ -11,14 +11,16 @@ type UserBottomNavProps = {
 const UserBottomNav: React.FC<UserBottomNavProps> = ({ currentPageId }) => {
   const { home, basket, orders } = pages;
 
+  const { token } = theme.useToken();
```

Comment:
What pattern to use? Should pattern be like
`const { token: { colorBgBase },} = theme.useToken();`
then we use colorBgBase not token.colorBgBase